cmd/tfimport/README.md

```diff
@@ -85,3 +85,4 @@ Terraform configs, including importing resources from within modules.
 - [google_compute_subnetwork](https://www.terraform.io/docs/providers/google/r/compute_subnetwork.html)
 - [google_compute_router](https://www.terraform.io/docs/providers/google/r/compute_router.html)
 - [google_compute_router_nat](https://www.terraform.io/docs/providers/google/r/compute_router_nat.html)
+- [google_dns_record_set](https://www.terraform.io/docs/providers/google/r/dns_record_set.html)
```

Comment:
This list is getting pretty large too, so when you group resources by top level service consider breaking this list into subheadings too.

Reply:
Ack, made a note.